pkg/minikube/logs/logs.go

```diff
@@ -81,6 +81,8 @@ var importantPods = []string{
 	"kube-scheduler",
 	"kube-proxy",
 	"kube-controller-manager",
+	"kindnet",
+	"storage-provisioner",
```

Comment:
```suggestion
```
`storage-provisioner` is included in `enabledAddonPods` down below L301

Reply:
hm... thats odd in the Logs that I see it I only see one time 
```
Initializing the minikube storage provisioner
```
if this was added duplicate I should had seen two blocks of storage provisioner logs